### PR TITLE
{chem}[intel/2015a] Octopus 4.1.2

### PR DESCRIPTION
**Pull request by jordiblasco**
The source_urls of netCDF and netCDF-Fortran have been updated.
The versions of Cmake used by METIS and ParMETIS have been changed for software stack compatibility.

### CODE REVIEW

**Comment by boegel** [comment]
@jordiblasco: the netCDF URL issues have already been tackled via #1766 and #1767; you'll need to sync your branch with current `develop` and resolve the conflicts you run into

File: easybuild/easyconfigs/m/METIS/METIS-5.1.0-intel-2015a.eb [hunk 1]
```diff
@@ -17,6 +17,6 @@ source_urls = [
 
 patches = ['METIS_IDXTYPEWIDTH.patch']
 
-builddependencies = [('CMake', '3.2.1')]
+builddependencies = [('CMake', '3.0.0')]
```
Comment (boegel) [inline easybuild/easyconfigs/m/METIS/METIS-5.1.0-intel-2015a.eb]: please don't change dependency versions (let alone *downgrade* them), you may introduce conflicts higher up

File: easybuild/easyconfigs/p/ParMETIS/ParMETIS-4.0.3-intel-2015a.eb [hunk 2]
```diff
@@ -17,6 +17,6 @@ source_urls = [
 ]
 sources = [SOURCELOWER_TAR_GZ]
 
-builddependencies = [('CMake', '3.1.0', '', ('GCC', '4.9.2'))]
+builddependencies = [('CMake', '3.0.0')]
```
Comment (boegel) [inline easybuild/easyconfigs/p/ParMETIS/ParMETIS-4.0.3-intel-2015a.eb]: no changing/downgrading please

**Comment by boegel** [comment]
Jenkins: ok to test

**Comment by fgeorgatos** [comment]
fyi.
if you follow the link proposed over here:
https://github.com/hpcugent/easybuild-easyblocks/issues/83
you will end up in these proposed guidelines:
http://www.hpcadvisorycouncil.com/pdf/Octopus_Best_Practices.pdf
If you find something interesting there, please let us know! If you don't (go through), that's ok too :)

**Comment by boegel** [comment]
Octopus 5.0.0 got merged via #2643, is this still relevant?

**Comment by migueldiascosta** [comment]
@jordiblasco, this PR is being closed for the following reason(s): no activity for > 6 months, obsoleted by more recent PRs.
Please don't hesitate to reopen this PR or add a comment if you feel this contribution is still relevant.
For more information on our policy w.r.t. closing PRs, see https://easybuild.readthedocs.io/en/latest/Contributing.html#why-a-pull-request-may-be-closed-by-a-maintainer

**Pull request closed without merging**